recipes/si/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.1)
```
3.1 is enough

Reply:
Not for `cxx_std_17`, it requires cmake 3.8

Reply:
Indeed! But could be used set_property instead

Reply:
`CXX_STANDARD 17` also requires cmake 3.8+

Reply:
I think sticking with cmake 3.8. is the safer/cleaner way than doctoring around with the properties

Reply:
> `CXX_STANDARD 17` also requires cmake 3.8+

True! Let's keep 3.8